src/HOFs/propFormatter.js

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react'
+import { PARAMS_EXP_MAP } from "./constants"
+
+/**
+ * Creates a 1-step, or complete, URL from `domain` and `src` Strings.
+ *
+ * - First, the function checks if src has a defined `domain`. If it does, it 
+ * checks to see if `src` has a scheme, and prepends "http" or "https" as needed
```

Comment:
> and prepends "http" or "https"

Is there any situation where "http" is prepended? Based on L32 it looks like it will always be "https"

Reply:
Shoot nice catch. 

As of right now will always prepend `https` since the `useHttps` argument isn't being used.

Ideally, if the Provider and Consumer combined props have `useHttps` prop as `false`, this function would prepend `http`.

Reply:
Addressed in this commit https://github.com/imgix/react-imgix/pull/790/commits/311082c487c0e2b6ec20895edf53a95701fb1130

Reply:
Add tests in this commit https://github.com/imgix/react-imgix/pull/790/commits/595600cfa5bccde51f3c721fcaea5945a647257a